RELEASE.CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+### Release 2024-04-25
+* **Amazon.Lambda.Annotations (1.3.1)**
+  * Update User-Agent string
+* **Amazon.Lambda.TestTool (0.15.2)**
+  * Update User-Agent string
+* **AWSLambdaPSCore PowerShell Module (4.0.1.0)**
+  * Update User-Agent string
+* **Amazon.Lambda.Serialization.SystemTextJson (2.4.3)**
+  * Update ByteArrayConverter to support base64 deserialization of byte[]
```

Comment:
nit: I think it's nice to thank external contributors and link the PR like we did down below on 2024-01-18.

Reply:
Done